README.rst

```diff
@@ -81,7 +81,7 @@ It can be seen in action at the following sites:
 - Zeelandia GmbH & Co. KG: https://www.zeelandia.de (by kitconcept GmbH)
 - VHS-Ehrenamtsportal: https://vhs-ehrenamtsportal.de (by kitconcept GmbH)
 - German Physical Society: https://www.dpg-physik.de (by kitconcept GmbH)
-- Universitat Politècnica de Catalunya: https://www.upc.edu (by kitconcept GmbH)
+- Universitat Politècnica de Catalunya: https://www.upc.edu/ca (by kitconcept GmbH)
```

Comment:
@stevepiercy why would you want to point international visitors to the Catalan version of that site? If we want to avoid a redirect here we should rather use the EN version: https://www.upc.edu/en

Reply:
Correct. The default redirect is to `/ca`. I will change it to `/en`. Thank you!